cmd/swapcli/errors.go

```diff
@@ -5,9 +5,13 @@ import (
 )
 
 var (
-	errNoMinAmount      = errors.New("must provide non-zero --min-amount")
-	errNoMaxAmount      = errors.New("must provide non-zero --max-amount")
-	errNoExchangeRate   = errors.New("must provide non-zero --exchange-rate")
-	errNoProvidesAmount = errors.New("must provide non-zero --provides-amount")
-	errNoDuration       = errors.New("must provide non-zero --duration")
+	errNoMinAmount                        = errors.New("must provide non-zero --min-amount")
+	errNoMaxAmount                        = errors.New("must provide non-zero --max-amount")
+	errNoExchangeRate                     = errors.New("must provide non-zero --exchange-rate")
+	errNoProvidesAmount                   = errors.New("must provide non-zero --provides-amount")
```

Comment:
You could use the flag name constants when creating these strings.  Then it will be easier to update the flag names in the future. There probably isn't a way to fix this, but if someone came up with a good, yet concise, way to make the coin type and numerator/denominator types obvious from these flag names, it would be a definite use experience win.

Reply:
updated!